projects/angular/src/forms/common/providers/ng-control.service.ts

```diff
@@ -31,11 +34,33 @@ export class NgControlService {
     return this._helpers.asObservable();
   }
 
+  get additionalControlsChanges(): Observable<NgControl[]> {
+    return this._additionalControlsChanges.asObservable();
+  }
+
+  get hasAdditionalControls() {
+    return !!this._additionalControls?.length;
+  }
+
   setControl(control: NgControl) {
+    this._control = control;
     this._controlChanges.next(control);
   }
 
   setHelpers(state: Helpers) {
     this._helpers.next(state);
   }
+
+  addAdditionalControl(control: NgControl) {
+    this._additionalControls.push(control);
+    this._additionalControlsChanges.next(this._additionalControls);
+  }
+
+  getControl() {
+    return this._control;
+  }
+
+  getAdditionalControls() {
+    return this._additionalControls;
+  }
```

Comment:
```suggestion
  get control() {
    return this._control;
  }

  get additionalControls() {
    return this._additionalControls;
  }
```

These can be get accessors.